packages/public-api-fixture-server/fixtures/field.html.ejs

```diff
@@ -19,7 +19,8 @@
     }
 
     // Stub broker behavior
-    if (config().type === 'number') {
+    var tokenizeElementType = config().recurly.fields.tokenizeElement || 'number';
```

Comment:
I'm thinking it would be best to just assign this behavior onto the window, since the security profile used in specs is permissive enough to do something like the following:

**field.html.ejs**
```js
window.stubTokenizationElementName = 'number';

function setStubTokenizationElementName (name) {
  window.stubTokenizationElementName = name;
}

// ...

function onToken (body) {
  // ...
  if (stubTokenizationElementName === 'cvv') {
    recurly.config.required = ['cvv'];
  }
  // ...
}
```

**test**
```js
element.iframe.window.setStubTokenizationElementName('cvv')
```